engine/core/src/main/java/org/datacleaner/descriptors/RemoteTransformerDescriptor.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.descriptors;
+
+/**
+ * A {@link TransformerDescriptor} for remote transformers.
+ */
+public interface RemoteTransformerDescriptor extends TransformerDescriptor {
+    public void addPropertyDescriptor(ConfiguredPropertyDescriptor propertyDescriptor);
```

Comment:
Again I don't think mutating behaviour should be part of the interface. At least consider: Would anyone (except the provider which is effectively the factory of this object) have any use of setting/adding a property?